CONTRIBUTING.md

```diff
@@ -83,12 +83,16 @@ changes to the API (e.g., remove a public method) or to the ABI (e.g., change re
 * The project aims to provide a consistent experience across all the public APIs. It is important to ensure consistency (same look and feel) across different public packages.
 * Use `final` for public classes everywhere it is possible, this ensures that these classes cannot be extended when the API does not intend to offer that functionality.
 * In general, we use the following ordering of class members:
-    * static fields (final before non-final)
-    * non-static fields (final before non-final)
-    * constructors
-    * static methods
-    * instance methods
-    * inner classes
+    * Static fields (final before non-final)
+    * Instance fields (final before non-final)
+    * Constructors
+      * In static utility classes (where all members are static), the private constructor
+        (used to prevent construction) should be ordered after methods instead of before methods.
```

Comment:
should this be above or below "Nested classes" I wonder

Reply:
I personally put them above nested classes (still grouped with methods), but I don't mind changing if you prefer

Reply:
I don't know if I have much of an opinion, actually. Just thought I'd ask the question. :)